packages/hydrogen-demo-site/.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
I'd be more inclined to go with Prettier, but if this is what the hydrogen demo site ships with, I guess, go with it.